Sources/MapboxDirections/Incident.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+/**
+ `Incident` describes any corresponding event, used for annotating the route.
+ */
+public struct Incident: Codable, Equatable {
+
+    private enum CodingKeys: String, CodingKey {
+        case identifier = "id"
+        case type
+        case description = "description"
+        case creationTime = "creation_time"
+        case startTime = "start_time"
+        case endTime = "end_time"
+        case impact = "impact"
+        case subtype = "sub_type"
+        case subtypeDescription = "sub_type_description"
+        case alertCodes = "alertc_codes"
+        case lanesBlocked = "lanes_blocked"
+        case geometryIndexStart = "geometry_index_start"
+        case geometryIndexEnd = "geometry_index_end"
+    }
+
+    var identifier: String
+    var type: String
+    var description: String
+    var creationTime: String
+    var startTime: String
+    var endTime: String
+    var impact: String?
+    var subtype: String?
+    var subtypeDescription: String?
+    var alertCodes: [Int]
+    var lanesBlocked: [Int]
+    var geometryIndexStart: Int
+    var geometryIndexEnd: Int
```

Comment:
These properties are all internal, so they aren’t available to clients such as the navigation SDK.